hera_mc/tests/test_connections.py

```diff
@@ -96,6 +97,9 @@ def test_get_dossier(self):
         self.assertTrue(len(x) == 0)
         x = self.h.get_part_connection_dossier('test_part2', 'active', 'all', at_date=old_time, exact_match=True)
         self.assertTrue(len(x) == 0)
+        z = {}
+        z['tst'] = cm_handling.PartConnectionDossierEntry('test_part1', 'active', 'all', at_date='now')
+        self.h.show_connections(z)
```

Comment:
Testing print statements needs to be done by capturing the output and making sure that what it prints out is what you expect.

Reply:
https://stackoverflow.com/questions/4219717/how-to-assert-output-with-nosetest-unittest-in-python

Reply:
Example in hera_cal: https://github.com/HERA-Team/hera_cal/blob/master/hera_cal/tests/test_utils.py#L429

It uses the context manager defined at the top of that test file.